tests/helpers.py

```diff
@@ -2257,3 +2258,55 @@ def validate_pods_are_running_and_not_restarted(
     logger.error(f"Pod is in {pod_state} state and restart count of pod {restart_count}")
     logger.info(f"{pod_obj}")
     return False
+
+
+def delete_objs(pvc_objs):
+    """
+    Delete pvc objects
+    Args:
+        pvc_objs (list): pvc objects to delete
+    """
+    for obj in pvc_objs:
+        obj.delete()
+        obj.ocp.wait_for_delete(obj.name)
+
+
+def handle_threading(pvc_objs, target_parm, arg_parm=None):
```

Comment:
We do have threading in other tests like running IO in pod https://github.com/red-hat-storage/ocs-ci/blob/master/tests/e2e/scale/test_pv_scale_ocs_create_delete_pvcs.py#L75
If this thread part works specific to pvc then would request to change function name accordingly

Reply:
Okay. I will rename this function